### ISSUE TITLE
Return badge from the Render() as a []byte instead of writing it directly to io.Writter

### ISSUE THREAD

**Issue by narqo** [narqo/go-badge#4]
This sounds very personal, but to have an API like the one below looks more friendly.

```
var b []byte
b = Render("subj", "stat", "red")
```

For example, this should make usage of external SVG minifier ([svgo](https://github.com/svg/svgo) for go anyone?) more easier.

**Comment by jgautheron** [comment]
Yes definitely, that's how I'd like to use it.

``` go
b, err := badge.Generate("text", badge.ColorFoo)
```

**Comment by narqo** [comment]
Addressed with #11
